StarVMC/StarAgmlLib/AgMath.h

```diff
@@ -67,10 +69,10 @@ template <typename T> T min(const T &a, const T &b ){ return TMath::Min(a,b); }
 Int_t mod(const Int_t &a, const Int_t &b);
 Float_t mod(const Float_t &a, const Float_t &b);
 
-// #define cos(x) TMath::Cos( (Double_t) x )
-// #define sin(x) TMath::Sin( (Double_t) x )
-// #define tan(x) TMath::Tan( (Double_t) x )
-// #define exp(x) TMath::Exp( (Double_t) x )
+#define cos(x) TMath::Cos( (Double_t) x )
+#define sin(x) TMath::Sin( (Double_t) x )
+#define tan(x) TMath::Tan( (Double_t) x )
+#define exp(x) TMath::Exp( (Double_t) x )
```

Comment:
Wrapping these in the namespace will not lead to AgMath::sin(x) etc, if that's what you are expecting.
Can you remind me why you need to redefine these common trigonometric functions? Is AgMath.h included in some generated code?